ui.frontend/src/main/webpack/site/js/components/searchTabs.ts

```diff
@@ -78,6 +76,10 @@ const buildSearchTab = ({
     })
   })
 
+  searchTab.addEventListener('click', () => {
+    searchTab.dispatchEvent(onSwitchTabClick)
```

Comment:
**Thought**

I think there is a way to not need to repeat searchTab.dispatchEvent but use the argument within the EventListener. Eg. event.target

Reply:
This is done